roles/language/tasks/main.yml

```diff
@@ -1,9 +1,13 @@
 ---
-- name: install language package(s)
-  sudo: yes
-  apt: pkg={{ item }} state=latest force=yes
-  with_items: language_packages
+#- name: install language package(s)
+#  sudo: yes
+#  apt: pkg={{ item }} state=latest force=yes
+#  with_items: language_packages
 
+- name: generate locales
+  shell: "locale-gen {{ language }}"
+  sudo: yes
+  
```

Comment:
I also had to run in the terminal:

`export LANGUAGE="en_US.UTF-8"
echo 'LANGUAGE="en_US.UTF-8"' | sudo tee --append /etc/default/locale
echo 'LC_ALL="en_US.UTF-8"' | sudo tee --append /etc/default/locale`